extension/api.js

```diff
@@ -83,10 +88,20 @@
 			url += 'api/v3/notifications';
 		}
 
-		url += participating;
+		url += `?${query.join('&')}`;
 
 		xhr('GET', url, opts, (data, status, response) => {
```

Comment:
The problem is that as long as we only need notifications count, this is perfectly fine, be if we need some of that data (#66), we're in a bad situation because we will need to perform additional requests.